peps/pep-0735.rst

```diff
@@ -0,0 +1,650 @@
+
+Use of Dependency Groups
+------------------------
+
+Tools which support Dependency Groups are expected to provide new options and
+interfaces to allow users to install from Dependency Groups.
+
+No syntax is defined for expressing the dependency group of a package, for two
+reasons:
+
+* it would not be valid to refer to the dependency groups of a third-party
+  package from PyPI (because the data is defined to be unpublished)
+
+* there is not guaranteed to be a current package for dependency groups -- part
+  of their purpose is to support non-package projects
+
+For example, a possible pip interface for installing dependency groups
+would be:
+
+.. code:: shell
+
+    pip install --dependency-groups=test,typing
+
+Note that this is only an example. This PEP does not declare any requirements
+for how tools support the installation of Dependency Groups.
+
+Reference Implementation
+========================
+
+There is currently no reference implementation/consumer of this specification.
+
+Backwards Compatibility
+=======================
+
+At time of writing, the ``dependency-groups`` namespace within a
+``pyproject.toml`` file is unused. Since the top-level namespace is
+reserved for use only by standards specified at packaging.python.org,
+there should be no direct backwards compatibility concerns.
+
+Security Implications
+=====================
+
+This PEP introduces new syntaxes and data formats for specifying dependency
+information in projects. However, it does not introduce newly specified
+mechanisms for handling or resolving dependencies.
+
+It therefore does not carry security concerns other than those inherent in any
+tools which may already be used to install dependencies -- i.e. malicious
+dependencies may be specified here, just as they may be specified in
+``requirements.txt`` files.
+
+How to Teach This
+=================
+
+This feature should be referred to by its canonical name, "Dependency Groups".
+
+The basic form of usage should be taught as a variant on typical
+``requirements.txt`` data. Standard dependency specifiers (:pep:`508`) can be
+added to a named list. Rather than asking pip to install from a
+``requirements.txt`` file, either pip or a relevant workflow tool will install
+from a named Dependency Group.
+
+For new Python users, they may be taught directly to create a section in
+``pyproject.toml`` containing their dependency groups, similarly to how they
+are currently taught to use ``requirements.txt`` files.
+This also allows new Python users to learn about ``pyproject.toml`` files
+without needing to learn about package building.
+A ``pyproject.toml`` file with only ``[dependency-groups]`` and no other tables
+is valid.
+
+For both new and experienced users, the object style used in Dependency Object
+Specifiers will need to be explained. Support for
+``path = ".", editable = true`` and
+``path = ".", editable = true, extras = ["extra"]`` should be taught
```

Comment:
```suggestion
``{path = ".", editable = true}`` and
``{path = ".", editable = true, extras = ["extra"]}`` should be taught
```
I've seen this is used in a few other places in the text (not the examples), too.